portfolio/src/main/java/com/google/sps/controller/HttpRequestServlet.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(name = "httpRequestServlet",
+            urlPatterns = {"/httpRequestServlet"})
+public class HttpRequestServlet extends HttpServlet {
+
+    /**
+     * Handles HTTP GET requests.
+     *
+     * @param request
+     *         the HttpServletRequest object
+     * @param response
+     *         the HttpServletResponse object
+     *
+     * @throws ServletException
+     *         if there is a Servlet failure
+     * @throws IOException
+     *         if there is an IO failure
+     */
+    public void doGet(HttpServletRequest request,
+                      HttpServletResponse response) throws
+                                                                                ServletException,
+                                                                                IOException {
+        response.setContentType("text/html");
+
+        PrintWriter out = response.getWriter();
+        System.out.println("Is this logging? (HttpRequestServlet.java) (bad " +
+                           "log version)");
+        log("Is this logging? (HttpRequestServlet.java)");
+        out.print("<HTML>");
```

Comment:
nit - Any reason these are uppercase?

Reply:
I am not sure why I made these uppercase. I will keep tags in lowercase moving forward.